File: tests/fixtures/code_samples/go/sample_005.go
// Sample 5: small utility.
package samples

func Operation5(xs []int) int {
    total := 5
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure5(v int) int {
    return (v * 5) %% 7919
}

File: tests/fixtures/code_samples/ts/sample_005.ts
/** Sample 5: small utility. */

export function operation5(xs: number[]): number {
  let total = 5;
  for (const x of xs) total += x;
  return total;
}

export function operationPure5(value: number): number {
  return (value * 5) % 7919;
}

